instrumentation/google.golang.org/grpc/otelgrpc/test/grpc_test.go

```diff
@@ -37,6 +35,8 @@ import (
 	"go.opentelemetry.io/otel/sdk/trace"
 	"go.opentelemetry.io/otel/sdk/trace/tracetest"
 	semconv "go.opentelemetry.io/otel/semconv/v1.17.0"
+
+	pb "google.golang.org/grpc/interop/grpc_testing"
```

Comment:
is it OK for us to continue using this package?  Seems odd to be able to use this but not interop?

Reply:
The pb.go's are more unlikely to change ever and to cause a regression when an API used only for testing changes in the interop package. We do want to move the interop package to internal, but not in the near future. It might be better to generate pb.go's that are more specific for the tests here at some point to remove total dependency.